bin/build/build.sh

```diff
@@ -223,11 +150,11 @@ insert_space() {
 
 apply_pattern() {
     sed -e "s/%FilesMatchPattern%/$( \
-        cat "src/files_match_pattern" | \
+        cat "${repo_root}/src/files_match_pattern" | \
         sed '/^#/d' | \
         tr -s '[:space:]' '|' | \
         sed 's/|$//' \
-    )/g" --in-place "$1"
+    )/g" -i "" "$1"
```

Comment:
👍

Reply:
:-) When to best close that issue? Right now, or after this branch has been merged?

Reply:
Oh, closed already. Thx!